hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/MethodProbe.java

```diff
@@ -63,8 +64,10 @@ void register(MetricsRegistryImpl metricsRegistry, Object source, String namePre
     }
 
     private String getName(String namePrefix) {
-        String name = method.getName();
```

Comment:
we need to make sure that we are not going to generate litter when the diagnostics writes to e.g. file. I don't have the logic of this part in my head at the moment.